src/app/components/home/home.component.ts

```diff
@@ -38,4 +47,8 @@ export class HomeComponent {
   onSwipeLeft(trip: string) {
     console.log('Delete', trip);
```

Comment:
Dispatch here, you have the action already. Do a popconfirm or even a default browser confirm and then dispatch the delete.